source/sass/blocks/about.scss

```
@@ -14,7 +16,91 @@
 }
 
 .about__item {
-  padding-top: 45px;
+  position: relative;
+  padding-top: 60px;
+  padding-right: 35%;
 
-  @include fonts(700, 17px, 118%);
+  @include fonts(700, 17px, $color-general, 118%);
+
+  &::before {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 55px;
+    height: 40px;
+    background-position: 0 center;
+    background-repeat: no-repeat;
+  }
+
+
+  &--eco {
+    &::before {
+      background-image: url("../img/icon/icon-eco.svg");
+    }
+  }
```

```suggestion
  &--eco::before {
     background-image: url("../img/icon/icon-eco.svg");
  }
```
Эти модификаторы чисто для иконок, поэтому можно не делать вложенность